src/GameStore.js

```diff
@@ -36,32 +36,40 @@ export class GameStore {
         return this.heroes
     }
     toggleShowItems = () => {
-        this.showItems = !this.showItems;
-        this.showSkills = false
-        this.showActions = false
-        this.activeAction = null
-        this.activeSkill = null
-        this.activeItem = null
+        runInAction(() => {
```

Comment:
why do you need this in the store, isn't every function in the store an action?

Reply:
mobx complaining about mutation of state outside of actions since it's in strict mode.

Reply:
I don't know, I'm not familiar enough with React to know what the baseline functionality should be nor the best way to fix the issues. I was just trying to fix errors in the console and this seemed to do the trick.

Reply:
fair enough!